app/controllers/users_controller.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+class UsersController < ApplicationController
+  def index
+    @users = User.page(params[:page]).per(20)
```

Comment:
I would suggest to move all such controllers which requires user to be logged in into new namespace /acccount